index.js

```diff
@@ -474,7 +476,9 @@ function getMetadata (opts) {
       agent: {
         name: opts.agentName,
         version: opts.agentVersion
-      }
+      },
+      framework: undefined,
+      version: undefined
```

Comment:
shouldn't this be opts.serviceVersion ?

Reply:
Sorry, it's not immediately obvious why this is set to `undefined` here. It _is_ set to `opts.serviceVersion` later, if it has been provided. The reason for setting to `undefined` _here_ rather than just omitting it completely is a performance optimization related to the [hidden classes](https://richardartoul.github.io/jekyll/update/2015/04/26/hidden-classes.html) problem.